docs/release-notes/8.0.asciidoc

```diff
@@ -8,3 +8,35 @@
 [discrete]
 [[known-issues-8.0.0-rc2]]
 ==== Known issues
+
+*The Data view option might not display in upgraded environments with legacy alerts*
+
+To make the *Data view* option appear, a user with elevated role privileges must visit the {es-sec-app}, open a page that displays alert data (such as the Overview page), then refresh the page. ({pull}121390[#121390]).
+
+The role must have the following privileges:
+
+* *Cluster privileges*: The `manage` privilege
+* *Index privileges*: The `manage`, `write`,`read`, and `view_index_metadata` index privileges for the following system indices where `<kib-space>` is the {kib} space name:
+
+** `.siem-signals-<kib-space>`
+** `.lists-<kib-space>`
+** `.items-<kib-space>`
+** `.alerts-security.alerts-<kib-space>`
+** `.internal.alerts-security.alerts-<kib-space>-*`
+
+* *{kib} space*: `All` privileges for the `Security` feature (see
+{kibana-ref}/xpack-spaces.html#spaces-control-user-access[Feature access based on user privileges])
+
+NOTE: If new alerts are generated in an upgraded environment without legacy alerts, refreshing any page with alert data in {elastic-sec} will make the *Data view* option appear in the {es-sec-ui}.
+
+*Incorrectly activated rules may not generate alerts after upgrading to {stack} 8.0.0*
```

Comment:
I think "incorrectly activated rules" potentially can be confusing. Let's just simplify a bit. 
```suggestion
*Detection rules may not generate alerts after upgrading to {stack} 8.0.0*
```